java/s3/S3Uploads.java

```diff
@@ -256,7 +253,7 @@ private String bytesToHex(byte[] bytes) {
     private String base64EncodePolicy(JsonElement jsonElement) throws UnsupportedEncodingException
     {
         String policyJsonStr = jsonElement.toString();
-        String base64Encoded = (new BASE64Encoder()).encode(policyJsonStr.getBytes("UTF-8")).replaceAll("\n","").replaceAll("\r", "");
+        String base64Encoded = BinaryUtils.toBase64 (policyJsonStr.getBytes("UTF-8"));
```

Comment:
To answer my own question - this may not be needed (at least anymore). [An example in AWS' signature v2 docs](http://docs.aws.amazon.com/general/latest/gr/signature-version-2.html) leaves this out.